_posts/2019/2019-04-02-how-to-easily-manage-ssh-keys-part-2.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: how to easily manage ssh keys - part 2
```

Comment:
A new title would be good to showcase what this post is about. Maybe something with SSH, API and implementation :D

Reply:
I think its maybe not a bad option  to have the part 2 title since its clearly referencing thats a follow-up post of the other post. Having a completly new title may confuse readers. From your experience could this be a problem ? If we use another title it would be nice to reference the 2nd part in the first one i think.

If we want to use a new title here are some options:
- ssh key management API implementation
- how to easily manage ssh keys - API implementation
- implementing a ssh key management API

Reply:
I vote for option 2: How to easily manage SSH keys - API implementation

Reply:
:+1: